toolbox/githubctl/main.go

```diff
@@ -267,7 +270,8 @@ func main() {
 			log.Printf("Error during UpdateIstioVersionAfterReleaseTagsMadeOnDeps: %v\n", err)
 		}
 	case "uploadArtifacts":
-		if err := CreateIstioReleaseUploadArtifacts(); err != nil {
+		assertNotEmpty("ref_sha", refSHA)
```

Comment:
This works fine but we might want to move the assertion into subroutine (see line 216 217) to be consistent with the rest of the code.

Reply:
Done